Android build fails after upgrade from 202 to 203

### Description

Hi,
we encountered this build error on Android after last update. It seems to be related to recently added font manager. We use Skia in monorepo and there is lot of "paths" in error log so maybe this could be related to some bad paths in node_modules?

```
> Task :sentry_react-native:compileReleaseJavaWithJavac
Note: /home/runner/work/trezor-suite/trezor-suite/node_modules/@sentry/react-native/android/src/main/java/io/sentry/react/RNSentryModuleImpl.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :sentry_react-native:mergeReleaseGeneratedProguardFiles
> Task :sentry_react-native:mergeReleaseConsumerProguardFiles
> Task :sentry_react-native:syncReleaseLibJars
> Task :sentry_react-native:bundleReleaseLocalLintAar

> Task :shopify_react-native-skia:buildCMakeRelWithDebInfo[arm64-v8a]
C/C++: ninja: Entering directory `/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/android/.cxx/RelWithDebInfo/4gz4e2t3/arm64-v8a'
C/C++: In file included from /home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/rnskia/RNSkManager.cpp:8:
C/C++: In file included from ../../../../../cpp/api/JsiSkApi.h:44:
C/C++: ../../../../../cpp/api/JsiSkTypeFaceFontProviderFactory.h:10:10: fatal error: 'JsiSkTypefaceFontProvider.h' file not found

> Task :shopify_react-native-skia:buildCMakeRelWithDebInfo[arm64-v8a] FAILED

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':shopify_react-native-skia:buildCMakeRelWithDebInfo[arm64-v8a]'.
> com.android.ide.common.process.ProcessException: ninja: Entering directory `/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/android/.cxx/RelWithDebInfo/4gz4e2t3/arm64-v8a'
  [1/18] Building CXX object CMakeFiles/rnskia.dir/cpp/jni/JniPlatformContext.cpp.o
  [2/18] Building CXX object 'CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/jsi/JsiHostObject.cpp.o'
  [3/18] Building CXX object CMakeFiles/rnskia.dir/cpp/jni/JniSkiaManager.cpp.o
  [4/18] Building CXX object CMakeFiles/rnskia.dir/cpp/rnskia-android/SkiaOpenGLSurfaceFactory.cpp.o
  [5/18] Building CXX object 'CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/jsi/JsiValue.cpp.o'
  [6/18] Building CXX object CMakeFiles/rnskia.dir/cpp/rnskia-android/RNSkOpenGLCanvasProvider.cpp.o
  [7/18] Building CXX object 'CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/jsi/RuntimeAwareCache.cpp.o'
  [8/18] Building CXX object 'CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/jsi/RuntimeLifecycleMonitor.cpp.o'
  [9/18] Building CXX object 'CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/jsi/JsiPromises.cpp.o'
  [10/18] Building CXX object 'CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/rnskia/RNSkManager.cpp.o'
  FAILED: CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/rnskia/RNSkManager.cpp.o 
  /usr/local/lib/android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++ --target=aarch64-none-linux-android26 --gcc-toolchain=/usr/local/lib/android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64 --sysroot=/usr/local/lib/android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/sysroot -Drnskia_EXPORTS -I/home/runner/work/trezor-suite/trezor-suite/node_modules/react-native/ReactCommon/callinvoker -I/home/runner/work/trezor-suite/trezor-suite/node_modules/react-native/ReactCommon/jsi -I/home/runner/work/trezor-suite/trezor-suite/node_modules/react-native/ReactCommon -I/home/runner/work/trezor-suite/trezor-suite/node_modules/react-native/ReactCommon/react/nativemodule/core -I/home/runner/work/trezor-suite/trezor-suite/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/turbomodule/core/jni -I../../../../cpp/jni/include -I../../../../cpp/rnskia-android -I../../../../../cpp/skia/include/config -I../../../../../cpp/skia/include/core -I../../../../../cpp/skia/include/effects -I../../../../../cpp/skia/include/utils -I../../../../../cpp/skia/include/pathops -I../../../../../cpp/skia/modules -I../../../../../cpp/skia/modules/skparagraph/include -I../../../../../cpp/skia/include -I../../../../../cpp/skia -I../../../../../cpp/api -I../../../../../cpp/jsi -I../../../../../cpp/rnskia -I../../../../../cpp/rnskia/values -I../../../../../cpp/rnskia/dom -I../../../../../cpp/rnskia/dom/base -I../../../../../cpp/rnskia/dom/nodes -I../../../../../cpp/rnskia/dom/props -I../../../../../cpp/utils -isystem /home/runner/.gradle/caches/transforms-3/b515b38be608f5a762db9315db6d46ab/transformed/jetified-fbjni-0.3.0/prefab/modules/fbjni/include -isystem /home/runner/.gradle/caches/transforms-3/11657f31175e70366b4842da81b9a88d/transformed/jetified-react-android-0.71.8-release/prefab/modules/react_nativemodule_core/include -isystem /home/runner/.gradle/caches/transforms-3/11657f31175e70366b4842da81b9a88d/transformed/jetified-react-android-0.71.8-release/prefab/modules/jsi/include -isystem /home/runner/.gradle/caches/transforms-3/11657f31175e70366b4842da81b9a88d/transformed/jetified-react-android-0.71.8-release/prefab/modules/turbomodulejsijni/include -g -DANDROID -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security  -fexceptions -frtti -std=c++1y -DONANDROID -DSK_GL -DSK_GANESH -DSK_BUILD_FOR_ANDROID -DFOLLY_NO_CONFIG=1 -DFOLLY_HAVE_CLOCK_GETTIME=1 -DFOLLY_HAVE_MEMRCHR=1 -DFOLLY_USE_LIBCPP=1 -DFOLLY_MOBILE=1 -DON_ANDROID -DONANDROID -O2 -g -DNDEBUG -fPIC -std=gnu++17 -MD -MT 'CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/rnskia/RNSkManager.cpp.o' -MF CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/rnskia/RNSkManager.cpp.o.d -o 'CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/rnskia/RNSkManager.cpp.o' -c '/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/rnskia/RNSkManager.cpp'
  In file included from /home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/rnskia/RNSkManager.cpp:8:
  In file included from ../../../../../cpp/api/JsiSkApi.h:44:
  ../../../../../cpp/api/JsiSkTypeFaceFontProviderFactory.h:10:10: fatal error: 'JsiSkTypefaceFontProvider.h' file not found
  #include "JsiSkTypefaceFontProvider.h"
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  1 error generated.
  [11/18] Building CXX object CMakeFiles/rnskia.dir/cpp/jni/JniLoad.cpp.o
  [12/18] Building CXX object 'CMakeFiles/rnskia.dir/home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/cpp/rnskia/RNSkJsView.cpp.o'
  ninja: build stopped: subcommand failed.
  
  C++ build system [build] failed while executing:
      /usr/local/lib/android/sdk/cmake/3.18.1/bin/ninja \
        -C \
        /home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/android/.cxx/RelWithDebInfo/4gz4e2t3/arm64-v8a \
        rnskia
    from /home/runner/work/trezor-suite/trezor-suite/node_modules/@shopify/react-native-skia/android

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
java.lang.StackOverflowError (no error message)
```

### Version

0.1.203

### Steps to reproduce

Upgrade from 202 to 203 and run android production build.

### Snack, code example, screenshot, or link to a repository

-

--- a/package/cpp/api/JsiSkTypeFaceFontProviderFactory.h
+++ b/package/cpp/api/JsiSkTypeFaceFontProviderFactory.h
@@ -7,7 +7,7 @@
 
 #include "JsiSkData.h"
 #include "JsiSkHostObjects.h"
-#include "JsiSkTypefaceFontProvider.h"
+#include "JsiSkTypeFaceFontProvider.h"
 
 namespace RNSkia {
 

